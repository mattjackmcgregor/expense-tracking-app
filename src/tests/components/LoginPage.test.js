import React from 'react'
import {shallow} from 'enzyme'
import LoginPage from '../../components/LoginPage'
import toJSON from 'enzyme-to-json'

test('should render login component', () => {
  const wrapper = shallow(<LoginPage />)
  expect(toJSON(wrapper)).toMatchSnapshot()
})