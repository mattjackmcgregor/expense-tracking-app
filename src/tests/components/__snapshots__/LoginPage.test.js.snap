// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render login component 1`] = `
<div>
  <h1>
    login
  </h1>
</div>
`;
